packages/good-design/src/apps/bridge/MicroBridge.tsx

```diff
@@ -203,7 +203,7 @@ export const MicroBridge = ({
           >
             You will receive on {targetChain} 
           </Text>
-        <TokenOutput outputValue={expectedToReceive ?? '0'} />
+        <TokenOutput token="G$" outputValue={expectedToReceive ?? '0'} />
```

Comment:
```suggestion
        <TokenOutput outputValue={expectedToReceive ?? '0'} />
```

@L03TJ3 you have token = G$ as default prop